OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -69,6 +69,9 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(int clientID, byte[] data)
 		{
+			if (disposed)
+				return;
```

Comment:
As we talked about in IRC, I wouldn't be comfortable removing this piece of code until an upstream fix is implemented because otherwise it will crash every restart. Which would mean a broken PR